beetmoverscript/docker.d/worker.yml

```diff
@@ -99,6 +99,22 @@ bucket_config:
           buckets:
             geckoview: 'maven-default-s3-upload-bucket-1705kkmo2aj3c'
         dummy: {}
+      'COT_PRODUCT == "xpi" && (ENV == "dev" || ENV == "fake-prod")':
+        dep:
+          credentials:
+            id: { "$eval": "DEP_ID" }
+            key: { "$eval": "DEP_KEY" }
+          buckets:
+            xpi: 'net-mozaws-stage-delivery-archive'
+          url_prefix: 'https://ftp.stage.mozaws.net'
```

Comment:
We may want to add a `dummy: {}` line -- I'm not 100% sure but I think this lets us test basic sanity checks without actually pushing to a bucket. I *think* we have that for every pool, but we'd at least want the dev/fake-prod pools.

Reply:
OK. I added it.